src/TesApi.Web/BatchScheduler.cs

```diff
@@ -503,18 +607,51 @@ static IEnumerable<string> ConvertNodeErrorsToSystemLogItems(AzureBatchJobAndTas
                 return new CombinedBatchTaskInfo
                 {
                     BatchTaskState = BatchTaskState.MoreThanOneActiveJobFound,
-                    FailureReason = BatchTaskState.MoreThanOneActiveJobFound.ToString()
+                    FailureReason = BatchTaskState.MoreThanOneActiveJobFound.ToString(),
+                    Pool = azureBatchJobAndTaskState.Pool
                 };
             }
 
+            // Because a ComputeTask is not assigned to the compute node while the StartTask is running, IAzureProxy.GetBatchJobAndTaskStateAsync() does not see start task failures. Attempt to deal with that here.
```

Comment:
"attempt to deal with that here" - are there known reasons we won't be able to deal with it?

Reply:
"Attempt to" was correct for most of the time this comment has been here. I'll update the comment.